pkg/xds/topology/outbound.go

```diff
@@ -26,6 +26,8 @@ import (
 	envoy_tags "github.com/kumahq/kuma/pkg/xds/envoy/tags"
 )
 
+var log = core.Log.WithName("xds")
```

Comment:
Should we add some more informations here, `core.Log.WithName("xds").WithName("outbound")`

Reply:
just be careful with this since it's a package level logger, we might have to use `outboundLog` i.e. one per file or create them in the components

Reply:
we can, updated